module/Database/view/database/meeting/installform.phtml

```diff
@@ -120,6 +125,55 @@ $(document).ready(function () {
                 $('#install-organ').prop('disabled', true);
 
                 // enable member change events
+                $('button.reappoint').click(function(e)
+                {
+                    var element = $(e.target).parent().parent();
+                    var name = element.find('.name').html();
+                    $('#install-modal-reappoint div.modal-body span.name').html(name);
+
+                    // Add all special functions to the modal.
+                    var reappointFuctionForm = $('#install-modal-reappoint div.modal-body #reappoint-functions-form');
```

Comment:
```suggestion
                    var reappointFuctionForm = $('#install-modal-reappoint div.modal-body #reappoint-functions-form');
                    reappointFuctionForm[0].innerHTML = '';
```

Inserting a line here causes expected behaviour

![image](https://github.com/GEWIS/gewisdb/assets/17340320/897b8621-36e8-4c41-ba88-6d4dd7302431)

Reply:
Fixed in 0dc2bdbcd95ae9ecfea9fc577fb42d34a6f80678.